Gems/Atom/Feature/Common/Code/Source/PostProcess/PostProcessFeatureProcessor.cpp

```diff
@@ -45,6 +50,16 @@ namespace AZ
             m_deltaTime = deltaTime.count();
         }
 
+        void PostProcessFeatureProcessor::SetViewAlias(const AZ::RPI::ViewPtr sourceView, const AZ::RPI::ViewPtr targetView)
+        {
+            m_viewAliasMap[sourceView.get()] = targetView.get();
```

Comment:
question: Do we want to check if the entry already exists with a `find` call on the map, or is it always fine to add/update when calling this?

Reply:
I think it's fine to just overwrite it as a "set" function. The code will work as expected as well